client/transaction-pool/graph/src/listener.rs

```diff
@@ -134,6 +134,7 @@ impl<H: hash::Hash + traits::Member + Serialize, C: ChainApi> Listener<H, C> {
 	pub fn finalized(&mut self, block_hash: BlockHash<C>) {
 		if let Some(hashes) = self.finality_watchers.remove(&block_hash) {
 			for hash in hashes {
+				log::debug!(target: "txpool", "Sent finalization event for transaction {:?} at block {:?}", hash, block_hash);
```

Comment:
I think all other logs that include transaction hash are in form:
```
[<hash>] Something something
```
I tried to keep it consistent cause it helps processing the logs if needed.